lib/iterator.js

```diff
@@ -199,6 +199,66 @@ class Iterator {
     return result + suffix;
   }
 
+  // Find value in this iterator by comparing every value with
+  // the found one using `comparator`
+  // Signature: comparator[, accessor[, thisArg]]
+  //   comparator <Function> returns true if new value should be accepted
```

Comment:
```suggestion
  //   comparator <Function> returns `true` if new value should be accepted
```
For consistency with Returns.